java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/BinarySymbolicValue.java

```diff
@@ -67,4 +76,29 @@ public void computedFrom(List<SymbolicValue> symbolicValues) {
     return ImmutableList.of(programState);
   }
 
+  private List<ProgramState> handleBooleanWrapperWhenNull(SymbolicValue to, ProgramState programState, BooleanConstraint booleanConstraint) {
+    if (programState.getConstraint(to) == null) {
+      return ImmutableList.<ProgramState>builder()
+        .addAll(to.setConstraint(programState, BooleanConstraint.TRUE))
+        .addAll(to.setConstraint(programState, BooleanConstraint.FALSE)).build();
+    }
+    return shouldNotInverse().equals(booleanConstraint) ? ImmutableList.of(programState) : ImmutableList.of();
+  }
+
+  private List<ProgramState> handleBooleanWrapperWhenBoolean(SymbolicValue to, ProgramState programState, BooleanConstraint booleanConstraint, List<ProgramState> newProgramStates) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00103)